specification/cognitiveservices/data-plane/Language/preview/2022-07-01-preview/analyzetext.json

```diff
@@ -0,0 +1,2653 @@
+      "type": "object",
+      "description": "Supported parameters for a Custom Single Classification task.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/CustomTaskParameters"
+        }
+      ]
+    },
+    "CustomSingleLabelClassificationLROTask": {
+      "type": "object",
+      "description": "Use custom models to classify text into single label taxonomy",
+      "properties": {
+        "parameters": {
+          "$ref": "#/definitions/CustomSingleLabelClassificationTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextLROTask"
+        }
+      ],
+      "x-ms-discriminator-value": "CustomSingleLabelClassification"
+    },
+    "CustomLabelClassificationResult": {
+      "type": "object",
+      "properties": {
+        "documents": {
+          "type": "array",
+          "description": "Response by document",
+          "items": {
+            "allOf": [
+              {
+                "$ref": "#/definitions/ClassificationDocumentResult"
+              }
+            ]
+          }
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/CustomResult"
+        }
+      ],
+      "required": [
+        "documents"
+      ]
+    },
+    "ClassificationResult": {
+      "type": "object",
+      "required": [
+        "category",
+        "confidenceScore"
+      ],
+      "properties": {
+        "category": {
+          "type": "string",
+          "description": "Classification type."
+        },
+        "confidenceScore": {
+          "type": "number",
+          "format": "double",
+          "description": "Confidence score between 0 and 1 of the recognized class."
+        }
+      }
+    },
+    "CustomMultiLabelClassificationTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for a Custom Multi Classification task.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/CustomTaskParameters"
+        }
+      ]
+    },
+    "CustomMultiLabelClassificationLROTask": {
+      "type": "object",
+      "description": "Use custom models to classify text into multi label taxonomy",
+      "properties": {
+        "parameters": {
+          "$ref": "#/definitions/CustomMultiLabelClassificationTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextLROTask"
+        }
+      ],
+      "x-ms-discriminator-value": "CustomMultiLabelClassification"
+    },
+    "ClassificationDocumentResult": {
+      "type": "object",
+      "properties": {
+        "class": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ClassificationResult"
+          }
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/DocumentResult"
+        }
+      ],
+      "required": [
+        "class"
+      ]
+    },
+    "HealthcareTaskParameters": {
```

Comment:
the FHIR feature (fhirVersion, fhirBundle) was present in 2022-05-15-preview but appears to be missing in this version

Reply:
ack - will add this to the PR.